thirdparty/linuxbsd_headers/alsa/asoundlib.h

```diff
@@ -38,7 +38,11 @@
 #include <poll.h>
 #include <errno.h>
 #include <stdarg.h>
+#ifdef __FreeBSD__
+#include <sys/endian.h>
+#else
 #include <endian.h>
+#endif // __FreeBSD__
```

Comment:
> There are minor thirdparty code modifications, hopefully its acceptable.

Usually, we use a patch system, we don't edit thirdparty code directly. See other thirdparty code directories.

```suggestion
#include <endian.h>
```

Reply:
The patch is to document the change made to thirdparty code, but the change should still be committed (so accepting your code suggestion would undo the fix here).

Reply:
@MBCX You need to remove the change here though.